rd_ui/app/app_layout.html

```diff
@@ -28,7 +28,18 @@
     <link rel="icon" type="image/png" sizes="32x32" href="/images/favicon-32x32.png">
     <link rel="icon" type="image/png" sizes="96x96" href="/images/favicon-96x96.png">
     <link rel="icon" type="image/png" sizes="16x16" href="/images/favicon-16x16.png">
+{% if ga_tracking_code %}
+  <script>
+  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
 
+  ga('create', '{{ga_tracking_code}}', 'auto');
+  ga('send', 'pageview');
```

Comment:
This will *not* be called on every page change, but only on the first load (like tab open or actual page refresh with Cmd+R). This is the problem I mentioned on Slack.